lib/web/apiserver.go

```diff
@@ -2107,6 +2111,36 @@ func (h *Handler) siteSessionsGet(w http.ResponseWriter, r *http.Request, p http
 	return siteSessionsGetResponse{Sessions: sessions}, nil
 }
 
+type siteTrackersGetResponse struct {
+	Trackers []*types.SessionTrackerV1 `json:"sessions"`
+}
+
+// siteTrackersGet gets the list of active site session trackers.
+//
+// GET /v1/webapi/sites/:site/namespaces/:namespace/trackers
+func (h *Handler) siteTrackersGet(w http.ResponseWriter, r *http.Request, p httprouter.Params, ctx *SessionContext, site reversetunnel.RemoteSite) (interface{}, error) {
+	clt, err := ctx.GetUserClient(site)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	items, err := clt.GetActiveSessionTrackers(r.Context())
```

Comment:
Any concerns with scale here? How will this behave with thousands or more active sessions?

cc @rosstimothy

Reply:
Let's also wait for @rosstimothy but I wrote all of the session tracker client and backend service code so I feel semi-qualified to at least give some context here. This currently incurs about the same load as a request via the old system to the auth which isn't too bad. One potential improvement would be to paginate this if it's large but that depends on what @rosstimothy has to say. Would also require some mid-level frontend surgery I think.

Reply:
Yeah these are fair points. We don't need to worry about the max grpc message size since trackers are sent over a stream from auth to proxy to specifically avoid this. Load times could perhaps become an issue in a large cluster, but it  shouldn't get worse than it already is (since it does the same thing). The option would be to plan for the future a bit but this would involve reworking some fetch API's to allow for pagination which is a bit tricky for non append only things (like audit events).

Reply:
We don't do any conversion that buffers in these routes, we mostly return the trackers raw as is to the frontend. This is the reason for using new routes instead of just retrofitting the new API into the old ones. Once this is approved I'm going to file a PR that changes the webapi to consume these new routes and the necessary changes to allow it to handle tracker data.